packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -23,17 +23,55 @@ import {
 	UsageError,
 	createChildMonitoringContext,
 } from "@fluidframework/telemetry-utils";
-import { IPendingContainerState } from "./container.js";
 import { ISerializableBlobContents, getBlobContentsFromTree } from "./containerStorageAdapter.js";
 
+export interface SnapshotWithBlobs {
+	/**
+	 * Snapshot from which container initially loaded.
+	 */
+	snapshotTree: ISnapshotTree;
```

Comment:
```suggestion
	baseSnapshot: ISnapshotTree;
```
you can't change the property name for pending state

Reply:
what's the issue on doing that if we change the name on all places where it's being read?

Reply:
The interface is being used internally and the feature is not public so there shouldn't be any problems for backward compatibility.

Reply:
any exist usage of the serialized data will break, like how this is breaking the snapshot tests